NotificationHubs/src/com/windowsazure/messaging/NotificationHubClient.java

```diff
@@ -0,0 +1,12 @@
+//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//----------------------------------------------------------------
+
+package com.windowsazure.messaging;
+
+/**
+ * This interface represents all actions that can be done on an Azure
+ * Notification Hub.
+ */
+public interface NotificationHubClient extends INotificationHub {
```

Comment:
Yes, because this is an eyesore and not consistent with Java coding standards, so creating this interface leads to the idea that the other may be removed at some point.